src/main/java/com/kodilla/ecommercee/controller/ProductController.java

```diff
@@ -0,0 +1,40 @@
+package com.kodilla.ecommercee.controller;
+
+import com.kodilla.ecommercee.domain.dto.ProductDto;
+import lombok.RequiredArgsConstructor;
+import org.springframework.web.bind.annotation.*;
+
+import java.math.BigDecimal;
+import java.util.ArrayList;
+import java.util.List;
+
+@RestController
+@RequestMapping("v1/shop/products")
+@RequiredArgsConstructor
+public class ProductController {
+
+    @GetMapping
+    public List<ProductDto> getAllProducts() {
+        return new ArrayList<>();
+    }
+
+    @GetMapping("/{id}")
```

Comment:
productId

Reply:
Ścieżka dostępu jest już `@RequestMapping("v1/shop/products")`, więc jak dla mnie nie warto powielać bo wiemy czego tyczy się `id`. W Product i ProductDto używam `Long id`

Reply:
Niekoniecznie, id mamy wiele, jak będziemy rozbudowywać aplikację, możemy się pogubić. W aplikacji Task mieliśmy tylko jedną tabelę a mimo to konkretnie wskazywaliśmy jakiego id używamy.